index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="menu-wrap">
+        <nav class="nav-bar">
+            <a href="#" class="brand">My Logo</a>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+        </nav>
+      </header>
+   <section class="wrapper">
+     <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
+     <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
+      <a href="#" class="connect">LET'S CONNECT</a><br>
+      <div class="socials">
+      <ul>
+        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+        <li><a href=""><img src="./images/linkedin.png" alt=""></a></li>
+        <li><a href=""><img src="./images/m.png" alt=""></a></li>
+        <li><a href=""><img src="./images/github.png" alt=""></a></li>
+        <li><a href=""><img src="./images/v.png" alt=""></a></li>
```

Comment:
- [OPTIONAL] Kindly give the `alt` attr a descriptive text because this will increase your website's accessibility by screen readers a clear idea of what the image is about (maybe something like a twitter link...etc)